sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/OtlpConfigUtil.java

```diff
@@ -171,6 +171,30 @@ private static URL validateEndpoint(@Nullable String endpoint, boolean allowPath
     return endpointUrl;
   }
 
+  private static byte[] readPemByConfig(ConfigProperties config, String prefix, String dataType, String suffix ) {
```

Comment:
`readFileBytes(...)`?

Reply:
not sure if understood. suggesting method rename? done.